src/Entities/RulesEngine/Actions/SendEmail/EmailNotification.php

```diff
@@ -0,0 +1,257 @@
+     *
+     * @return $this
+     */
+    public function setCc($value): self
+    {
+        return $this->setAttribute('cc', $value);
+    }
+
+    /**
+     * @return string[]|array
+     */
+    public function getBcc(): array
+    {
+        return $this->getAttribute('bcc');
+    }
+
+    /**
+     * @param string[]|array $value
+     *
+     * @return $this
+     */
+    public function setBcc($value): self
+    {
+        return $this->setAttribute('bcc', $value);
+    }
+
+    /**
+     * @return string
+     */
+    public function getSubject(): string
+    {
+        return $this->getAttribute('subject');
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function setSubject($value): self
+    {
+        return $this->setAttribute('subject', $value);
+    }
+
+    /**
+     * @return string
+     */
+    public function getText(): string
+    {
+        return $this->getAttribute('text');
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function setText($value): self
+    {
+        return $this->setAttribute('text', $value);
+    }
+
+    /**
+     * @return string
+     */
+    public function getHtml(): string
+    {
+        return $this->getAttribute('html');
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function setHtml($value): self
+    {
+        return $this->setAttribute('html', $value);
+    }
+
+    /**
+     * @return string
+     */
+    public function getEditor(): string
+    {
+        return $this->getAttribute('editor');
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function setEditor($value): self
+    {
+        return $this->setAttribute('editor', $value);
+    }
+
+    /**
+     * @return ResourceAttachment[]|array
+     */
+    public function getAttachments(): array
+    {
+        return $this->getAttribute('attachments');
+    }
+
+    /**
+     * @param ResourceAttachment[]|array $value
+     *
+     * @return $this
+     */
+    public function setAttachments(array $value): self
+    {
+        return $this->setAttribute('attachments', $value);
+    }
+
+    /**
+     * @param ResourceAttachment[]|array $value
+     *
+     * @return ResourceAttachment[]|array
+     */
+    public function createAttachments(array $value): array
+    {
+        return array_map(function ($data) {
+            return new ResourceAttachment($data);
+        }, $value);
+    }
+
+    /**
+     * @return int
+     */
+    public function getWeight(): int
+    {
+        return $this->getAttribute('weight');
+    }
+
+    /**
+     * @param int $value
+     *
+     * @return $this
+     */
+    public function setWeight($value): self
+    {
+        return $this->setAttribute('weight', $value);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function actionName(): string
+    {
+        return self::NAME_SEND_EMAIL;
```

Comment:
is this name correct?

Reply:
https://github.com/Rebilly/backend/blob/ed5d6aa41454165042b792c0cbef41e000ba3620/src/Rules/Domain/Action.php#L49